docker/web/development/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM php:5.6-fpm as gewisweb_web_development
```

Comment:
We could just make this an additional stage in the production docker file, saves a lot of code duplicaiton.

Reply:
The development version has all tools like npm and composer installed. It makes little sense to install those in production, then copy some stuff over, and then install them again.